code/modules/mob/living/simple_animal/hostile/bosses/clockmaster/phase_two_attacks.dm

```diff
@@ -0,0 +1,152 @@
+//Bunkers up wherever standing, charging up an AoE attack to blast anyone who stays too close. Causes hallucations and brain damage to those affected.
+/datum/action/boss/brain_blast
+	name = "Sevtug's Wrath"
+	icon_icon = 'icons/mob/actions/actions_minor_antag.dmi'
+	button_icon_state = "mimic_summon"
+	usage_probability = 15
+	boss_cost = 25
+	boss_type = /mob/living/simple_animal/hostile/boss/clockmaster/phase_two
+	say_when_triggered = "Let Sevtug consume you, feel his presence wipe your heretical conscious clear!"
+	var/aoe_radius = 3
+
+/datum/action/boss/brain_blast/IsAvailable()
+	. = ..()
+	if(!.)
+		return FALSE
+	if(boss.mid_ability)
+		return FALSE
+	return TRUE
+
+/datum/action/boss/brain_blast/Trigger(trigger_flags)
+	if(..())
+		boss.mid_ability = TRUE
+		boss.prevent_goto_movement = TRUE
+		var/mob/living/simple_animal/armored_individual = boss
+		armored_individual.damage_coeff = list(BRUTE = 0.5, BURN = 0.5, TOX = 0.5, CLONE = 0.5, STAMINA = 0, OXY = 0.5)
+		for(var/turf/target_tile as anything in RANGE_TURFS(aoe_radius, boss))
+			if(!(locate(/obj/effect/clockmaster_aoe_warning) in target_tile) && !(locate(/mob/living/simple_animal/hostile/boss/clockmaster/phase_two) in target_tile))
+				new /obj/effect/clockmaster_aoe_warning(target_tile)
+			addtimer(CALLBACK(src, .proc/BlastEm), 4 SECONDS)
+
+/datum/action/boss/brain_blast/proc/BlastEm()
+	var/mob/living/simple_animal/armored_individual = boss
+	armored_individual.damage_coeff = list(BRUTE = 1, BURN = 1, TOX = 1, CLONE = 1, STAMINA = 0, OXY = 1)
+	boss.mid_ability = FALSE
+	boss.prevent_goto_movement = FALSE
+	playsound(boss, 'sound/misc/fart.ogg', 25)
```

Comment:
Phenomenal

Reply:
listen i added this to amuse myself while testing and forgot to remove it